### ISSUE TITLE
Markup request body

### ISSUE THREAD

**Issue by deceze** [deceze/Sphinx-HTTP-domain#10]
Add possibility to markup the request body. Can be very similar to `http:response::` I suppose. In fact, `http:response::` could probably be reused or renamed to `http:body::` or similar.

**Comment by felciano** [comment]
Any news on getting this implemented?

**Comment by deceze** [comment]
This project may be considered abandoned. There's a more official HTTP domain available now as part of the official Sphinx extensions: http://packages.python.org/sphinxcontrib-httpdomain/

**Comment by felciano** [comment]
Yes I saw that. Unfortunately it is lacking some of the features your project seems to have (e.g. marking of optional parameters) so I was hoping yours was still active!

**Comment by deceze** [comment]
Sorry, not really. Maybe someday in the future, but currently I have no interest or time. I'd recommend you take this up with the maintainer of the official HTTP domain and/or the Sphinx community as a feature request.

**Comment by felciano** [comment]
OK thanks! OK to close this issue from my standpoint

**Comment by dadoeyad** [comment]
It should be written in the README that this project is obsolete.
